src/trans/gpu/internal/ftdir_mod.F90

```diff
@@ -77,7 +94,12 @@ SUBROUTINE FTDIR(ALLOCATOR,HFTDIR,PREEL_REAL,PREEL_COMPLEX,KFIELD)
 
     INTEGER(KIND=JPIM) :: KGL
 
+#ifdef IN_PLACE_FFT
     PREEL_COMPLEX => PREEL_REAL
+#else
+    CALL ASSIGN_PTR(PREEL_COMPLEX, GET_ALLOCATION(ALLOCATOR, HFTDIR%HREEL_COMPLEX),&
+      & 1_C_SIZE_T, int(KFIELD*D%NLENGTF*SIZEOF(PREEL_COMPLEX(1)),kind=c_size_t))
```

Comment:
```suggestion
      & 1_C_SIZE_T, INT(KFIELD*D%NLENGTF*SIZEOF(PREEL_COMPLEX(1)),KIND=CSIZE_T))
```
I know we are not very consistent everywhere, but I suggest not add more caps inconsistencies.